lib/redis/store/namespace.rb

```diff
@@ -154,6 +158,10 @@ def zrem(key, member)
         namespace(key) { |k| super(k, member) }
       end
 
+      if respond_to?(:ruby2_keywords, true)
```

Comment:
Is there any harm in depending on `ruby2_keywords` in the gemspec instead of checking whether the method is defined here? Looked up the gem and it seems useful for forward compatibility with Ruby 3...

Reply:
Not really any harm, but for having fixed keyword issues in many gems, I never saw the need for it, and I think most people just do the `respond_to?` check.

The gem is super tiny, so you can both consider it too small to justify adding a gem, or so small that it's not really a cost.

Long story short it's up to you. If you'd rather use the gem, I can change the PR.

Reply:
I'm alright with it either way, especially if others are doing the `respond_to?` check and that would look more familiar to those reading the source.